cli/popper/scm.py

```diff
@@ -183,6 +183,12 @@ def get_default_branch(r):
         else:
             url += '/'
 
+        auth_token = os.getenv('GITHUB_API_TOKEN')
+
+        if(auth_token is not None):
+            if('@' not in url):
+                url = url[:8]+auth_token+'@'+url[8:]
```

Comment:
can you add a comment explaining why 8 characters before and after`url[:8]`?

Reply:
Yeah sure.

Reply:
Actually I just realized that there can be other protocols defined http:// and git@ although git@ would automatically wouldn't be considered for adding token, I can add support for it too.

Reply:
@ivotron Should I also add a check that protocol should be HTTPS only because Token works only with HTTPS protocol.